css/style.css

```diff
@@ -33,41 +148,321 @@ body {
   text-transform: uppercase;
 }
 
-.product-item__name {
+.market-section__subtitle {
+  text-align: center;
+  margin: 51px 0 16px 0;
+}
+
+.products {
+  display: flex;
+  justify-content: space-between;
+}
+
+.products__product-card {
+  margin: 0 0 25px 0;
+  /* background-color: rgb(160, 32, 32); */ /* для проверки */
+}
+
+.product-card {
+  /* width: calc(100% / 4); */
+  width: 230px;
+  position: relative;
+  padding: 0 12px;
+  /* height: 261px;   */
+  /* border: 1px solid rgb(23, 125, 209); */
+}
+
+.product-card::after {
+  content: "";
+  position: absolute;
+  top: 40px;
+  right: 108%; /* Найти лучшее решение */
+  /* display: inline-block; */
+  width: 1px;
+  height: 100px;
+  background:  #000000;
+  opacity: 0.2;
+}
+
+.product-card:first-child::after {
+  display: none;
+}
+
+.product-card__picture {
+  margin: 0 0 26px 12px;
+  
+}
+
+.product-card__name {
+  display: inline-block;
   font-size: 12px;
   font-weight: 400;
+  margin-bottom: 5px;
+}
+
+.color-option {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.circles {
+  display: flex;
+}
+
+.circles__link {
+  width: 7px;
+  height: 7px;
+  border: 1;
+  border-radius: 50%;
+  background: #c4c4c4;
+  margin-left: 8px;
+  transition: .3s;
+}
+
+.circles__link--black {
+  background-color: #0e1211;
+}
+
+.circles__link--red {
+  background-color: #b62e3c;
+}
+.circles__link--yellow {
+  background-color: #fdb431;
+}
+
+.circles__link--orange {
+  background-color: #e29d69;
+}
+
+.circles__link--darkred {
+  background-color: #762e31;
 }
-.product-item__price {
+
+.circles__link--blue {
+  background-color: #41479b;
+}
+
+.circles__link--aqua {
+  background-color: #257da9;
+}
+
+.circles__link:last-child {
+  background-color: #fdb431;
+}
+
+.circles__link:hover {
+  transform: scale(1.5);
+}
+
+.product-card__price {
   font-size: 14px;
   font-weight: 700;
 }
 
+
+/* Promo-section */
+
+.promo-section {
+  padding: 24px 0 79px 0;
+}
+
+.video-block {
+  padding: 64px 33px;
+  display: flex;
+  /* align-items: center; */
+  /* max-width: 1030px; */
+  min-height: 495px;
+  background-color: #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654146)